### PR TITLE
Changed Clipboard dependency

### LINKED ISSUE #38: Security problems with `xcb`
There are a few security problems with package `xcb`. The version used in this project is vulnerable due to:
- https://github.com/mimuw-jnp2-rust/project-matrix-busters/security/dependabot/1
- https://github.com/mimuw-jnp2-rust/project-matrix-busters/security/dependabot/2
- https://github.com/mimuw-jnp2-rust/project-matrix-busters/security/dependabot/3
- https://github.com/mimuw-jnp2-rust/project-matrix-busters/security/dependabot/4
- https://github.com/mimuw-jnp2-rust/project-matrix-busters/security/dependabot/5
- https://github.com/mimuw-jnp2-rust/project-matrix-busters/security/dependabot/6

The issue exists because of [clipboard](https://crates.io/crates/clipboard) create that has not been updated for over four years.

Possible solutions:
a) fork the crate, update its dependencies and use forked version
b) implement such create inside the application

### PR DESCRIPTION
**Pull request by tudny**

### CODE REVIEW

**Comment by tudny** [comment]
Before it is merged it should be tested on:
- [x] Windows 
- [x] Ubuntu 
- [ ] Apple (if we have any)

**Comment by tudny** [comment]
DO NOT MERGE before JNP3 final grade

**Review by mgr0dzicki — APPROVED** [review]
Tested on Manjaro with Xfce. Good job!

**Pull request merged**
